frontend/src/views/WasteActionsPage/WasteActionPage.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div id="wasteaction-page">
+    <div v-if="wasteaction">
+      <BreadcrumbsNav :title="wasteaction.title" :links="[{ to: { name: 'WasteActionsHome' } }]" />
+      <v-row>
+        <img
+          :src="
+            wasteaction.lead_image
+              ? wasteaction.lead_image.meta.download_url
+              : '/static/images/wasteaction-default-image.jpg'
+          "
+          class="lead-image"
+          alt=""
+        />
+      </v-row>
+      <v-row>
+        <v-col cols="12" class="mt-8" sm="2">
+          <v-row justify="end">Type d'action</v-row>
```

Comment:
faut que les textes soient encapsuler dans des `p`

https://html.spec.whatwg.org/multipage/grouping-content.html#the-div-element
![Screenshot 2024-08-06 at 14-31-06 HTML Standard](https://github.com/user-attachments/assets/bd3d6cfb-982c-441a-a8ca-fde84e1f8cb9)